### PR TITLE
GitHub repositories

### PR DESCRIPTION
**Pull request by dairoquintero**

### CODE REVIEW

File: config/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -177,7 +177,7 @@
                      value="Type name ''{0}'' must match pattern ''{1}''."/>
         </module>
         <module name="MemberName">
-            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9]*$"/>
+            <property name="format" value="^[a-z][a-z0-9]*(_[a-zA-Z0-9]+)*$"/>
```
Comment (dairoquintero) [inline config/checkstyle/checkstyle.xml:180]: Add rule to enable names with  _ : example full_Name, needed to extract response in a class with this element

File: src/test/java/co/com/api/test/dto/GithubDto.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+@Getter
+@Setter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class GithubDto {
+  String id;
+  String name;
+  String full_Name;
+  String description;
+  String svn_url;
+  String default_Branch;
```
Comment (aperdomob) [inline src/test/java/co/com/api/test/dto/GithubDto.java]: add private

File: src/test/java/co/com/api/test/dto/GithubDto.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package co.com.api.test.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Data
```
Comment (aperdomob) [inline src/test/java/co/com/api/test/dto/GithubDto.java]: remove

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: src/test/java/co/com/api/test/dto/GithubDto.java (hunk 2 above)
Comment (dairoquintero) [reply src/test/java/co/com/api/test/dto/GithubDto.java]: Done

File: src/test/java/co/com/api/test/dto/GithubDto.java (hunk 3 above)
Comment (dairoquintero) [reply src/test/java/co/com/api/test/dto/GithubDto.java]: Done

**Review by dairoquintero — COMMENTED** [review]
Done

**Review by aperdomob — APPROVED** [review]

**Pull request merged**